cogs/guess.py

```diff
@@ -0,0 +1,93 @@
+from random import choice
+from collections import Counter, defaultdict, namedtuple
+import discord
+import os
+import asyncio
+import re
+"""Guessing game """
+class Guess:
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="guess")
+    async def mycom(self):
+        # welcoming the user
+        await self.bot.reply("guess what word I'm thinking of!")
+
+        # thinking
+        await asyncio.sleep(3)
+
+        # start guess
+        await self.bot.say("Start guessing...(hint: no caps)")
+        await asyncio.sleep(0.5)
+
+        # make secret
+        secret = "its gonna change"
+
+        # Probably need to create a word list and grab words from there
+        items = ["here", "are", "some", "strings", "of","which", "we", "will", 
+         "select", "one", "two", "three", "four", "five", "six", 
+         "seven", "eight", "nine", "ten", "python", "ruby", "java", "javascript", "traceback", "print", 
+         "random", "input", "raw", "print", "sleep", "time", "secret", "while", "then", "if", 
+         "loop", "in", "else", "break", "empty", "mocha", "matcha", "money", "spam","jazz","jam","spaz", 
+         "binder", "jokes", "chemistry", "person", "matcha", "night", "everyone", "pasta", "craving", 
+         "jolly", "ranch", "lit", "still", "first", "second", "third", "skull", "ketchup", 
+         "ice", "red", "life", "death", "magic", "comments", "class", "secret", "value", 
+         "variable", "guess", "incorrect", "russian", "roulette", "bot", "turn", 
+         "failed", "wood", "stone", "cobblestone", "blanket", "ore", "chest", "poster"]
+        #picks one word from list
+        secret = random.choice(items)
+
+        #creates a variable with an empty value
+        guesses = ''
+        turns = 5
+        while turns > 0:
+            failed = 0
+            for char in secret:
+                if char in guesses:
+                    await self.bot.say(char),
+                else:
+                    await self.bot.say("_"),
+                    failed += 1
+                #if failed is equal to zero
+                # Win text
+            if failed == 0:
+                await self.bot.say("You won! Good job! :aquaThumbsUp:")  
+                # exit the script"""
+            
+                break
+            # ask the user go guess a character"""
+            await self.bot.say("lower case letters please~~!")
+            guess = self.bot.discord.wait_for_message(timeout=None, author=None, channel=None, content=None, check=None)
+            # set the players guess to guesses"""
+            guesses += guess
+            # incorrect letter"""
+            if guess not in secret:
+                turns -= 1        
+                await self.bot.say("Wrong") 
+                #if the turns equal to zero
+                if turns == 0:
+                    # losing text
+                    await self.bot.say("You Lost, " +str(secret), " was the right word")
```

Comment:
Can we make this more obvious?
```
"You Lost, " +str(secret), " was the right word"
                         ^
```
The above is confusing in my opinion, since it looks like `" was the right word"` is another parameter.  Consider using `.format(...)` or `%` equivalent.